app/services/catalog_remediator.rb

```diff
@@ -49,20 +53,21 @@ def zipped_moab_versions_beyond_expiry
 
   def zipped_moab_versions_with_errors
     zipped_moab_versions_beyond_expiry.to_a.filter_map do |zipped_moab_version|
-      audit_results = audit_results_for(zipped_moab_version)
-      # If this check returns false, it indicates a lack of zip parts, in which
-      # case we still want to delete the ZMV, so short-circuit the rest of the
-      # block
+      audit_results = empty_audit_results(zipped_moab_version)
+      # when Audit::CatalogToArchive.check_child_zip_part_attributes() returns true, we have all ZipParts for the
+      #   ZippedMoabVersion, so continue to the next check
```

Comment:
I think I prefer the prior wording here but it could be because I probably wrote it. Would be good to get more @sul-dlss/infrastructure-team  👀 on this one.

Reply:
I tried to grok the previous wording, and I just couldn't (the unless is part of the problem) so this was one of the big reasons for this PR.